#gstreamer_pipeline: "alsasrc device=plughw:ArrayUAC10"
gstreamer_pipeline: "udpsrc port=12345 ! application/x-rtp,media=audio,clock-rate=16000,encoding-name=L16,channels=6 ! queue ! rtpL16depay ! audioconvert"
n_channels: 2
main_channel: 0
sample_rate: 16000
update_rate: 4.0
sound_intensity_threshold: 300
intensity_normalization_divider: 2000
microphone_frame_id: 'microphone_link'
stationary_frame_id: 'microphone_link'
doa:
    algorithm: 'none'
    #algorithm: 'NormMUSIC'
    #algorithm: 'SRP'
    #algorithm: 'FRIDA'
    #algorithm: 'MUSIC'
    #algorithm: 'TOPS'
    mic_coords: [ 0.0, 0.05, 0.0, 0.0, -0.05, 0.0 ]
    mic_channels: [ 0, 1 ]
    grid_type: '2d_linspace'
    azimuth: [ -90.0, 90.0, 32 ]
    nfft: 1024
vad:
    type: 'button'
detections:
    publish_detections: True
    publish_markers: True
    distance: 2.0
    2d_z_position: 1.0
debug_plot: True
intensity_histogram_forget_factor: 0.99
intensity_histogram_step: 100
